README.md

```diff
@@ -148,7 +149,33 @@ $ eregs build_from CFR-2012-title12-vol8-part1004.xml 12
 
 Here ```12``` is the CFR title number (in our case, for "Banks and Banking").
 
-Running the command will generate four folders, ```regulation```,
+
+#### Using `pipeline`
+```bash
+$ eregs pipeline title part an/output/directory
+```
+or
+```bash
+$ eregs pipeline title part https://yourserver/
+```
+
+Example:
+```bash
+$ eregs pipeline 27 447 /path/to/reg
+```
+
+`pipeline` pulls annual editions of regulations from the [Government Printing Office](http://www.gpo.gov/fdsys/browse/collectionCfr.action) and final rules from the [Federal Register](https://www.federalregister.gov/) based on the part that you give it.
+
+When you run `pipeline`, it:
+
+1. gets rules that exist for the regulation from the Federal Register API
+2. builds trees from annual editions of the regulation
+3. fills in any missing versions between annual versions by parsing final rules
+4. builds the layers for all these trees
+5. builds the diffs for all these trees, and
+6. writes the results to your output location
+
+Running either `pipeline` or `build_from` will generate four folders, ```regulation```,
```

Comment:
It might make sense to repeat this line in the two sections. The reader wouldn't need to jump over a section to read this, in that scenario. More importantly, `pipeline` doesn't write to the `OUTPUT_DIR`, it writes to the path specified when calling it (e.g. `an/output/directory` in the first listing)